demo/src/time.rs

```diff
@@ -90,10 +93,6 @@ impl TimeResource {
         }
     }
 
-    pub fn system_time(&self) -> &TimeContext {
```

Comment:
I wonder if it's worth making this `Option<TimeContext>` rather than removing it entirely.

Reply:
This is from an older project that was more "engine"-like and we're just using it as demo code. So I think if we don't use it, there isn't any reason to keep it around.